src/main/scala/xerial/sbt/pack/PackPlugin.scala

```diff
@@ -224,33 +225,51 @@ object PackPlugin extends AutoPlugin with PackArchive {
       out.log.info(logPrefix  + "Copying libraries to " + rpath(base, libDir))
       val libs: Seq[File] = packLibJars.value.map(_._1)
       out.log.info(logPrefix + "project jars:\n" + libs.map(path => rpath(base, path)).mkString("\n"))
-      libs.foreach(l => IO.copyFile(l, libDir / l.getName))
+      val projectJars = libs.map(l => {
+        val dest = libDir / l.getName
+        IO.copyFile(l, dest)
+        dest
+      })
 
       // Copy dependent jars
 
       val distinctDpJars = packModuleEntries.value
       out.log.info(logPrefix + "project dependencies:\n" + distinctDpJars.mkString("\n"))
       val jarNameConvention = packJarNameConvention.value
-      for (m <- distinctDpJars) {
+      val projectDepsJars = for (m <- distinctDpJars) yield {
         val targetFileName = resolveJarName(m, jarNameConvention)
-        IO.copyFile(m.file, libDir / targetFileName, true)
+        val dest = libDir / targetFileName
+        IO.copyFile(m.file, dest, true)
+        dest
       }
 
       // Copy unmanaged jars in ${baseDir}/lib folder
       out.log.info(logPrefix + "unmanaged dependencies:")
-      for ((m, projectRef) <- packAllUnmanagedJars.value; um <- m; f = um.data) {
+      val unmanagedDepsJars = for ((m, projectRef) <- packAllUnmanagedJars.value; um <- m; f = um.data) yield {
         out.log.info(f.getPath)
-        IO.copyFile(f, libDir / f.getName, true)
+        val dest = libDir / f.getName
+        IO.copyFile(f, dest, true)
+        dest
       }
 
       // Copy explicitly added dependencies
       val mapped: Seq[(File, String)] = (mappings in pack).value
       out.log.info(logPrefix + "explicit dependencies:")
-      for ((file, path) <- mapped) {
+      val explicitDepsJars = for ((file, path) <- mapped) yield {
         out.log.info(file.getPath)
-        IO.copyFile(file, distDir / path, true)
+        val dest = distDir / path
+        IO.copyFile(file, dest, true)
+        dest
       }
 
+      // put the list of jars in a file
+      val bw = new BufferedWriter(new FileWriter(libDir / "jars.mf"))
```

Comment:
Can we make it an optional behavior? For example, if `packJarListFile := Some(...)` is defined, generate this file. If not, skip this part.

Reply:
All right. This is done.